### PR TITLE
[InterventionalRadiologyController] Remove unused methods and redondant Data d_curvAbs

### PR DESCRIPTION
**Pull request by epernod**
Add getter to the member vector m_nodeCurvAbs to replace the Data d_curvAbs

Depends on PR https://github.com/sofa-framework/BeamAdapter/pull/60

### CODE REVIEW

File: src/BeamAdapter/component/controller/InterventionalRadiologyController.inl [hunk 1]
```diff
@@ -161,7 +160,7 @@ void InterventionalRadiologyController<DataTypes>::init()
 
     Inherit::init();
 
-    reinit();
+    this->reinit();
```
Comment (fredroy) [inline src/BeamAdapter/component/controller/InterventionalRadiologyController.inl:163]:
```suggestion
```
As reinit() was doing nothing before (even overriding the inherited class to force doing nothing actually), you can remove this line I think ?

File: src/BeamAdapter/component/controller/InterventionalRadiologyController.inl [hunk 2]
```diff
@@ -205,18 +204,7 @@ void InterventionalRadiologyController<DataTypes>::bwdInit()
...
-template <class DataTypes>
-void InterventionalRadiologyController<DataTypes>::reinit()
-{
-}
-
-
-template <class DataTypes>
-void InterventionalRadiologyController<DataTypes>::computeVertexT()
-{
+    this->reinit();
```
Comment (fredroy) [inline src/BeamAdapter/component/controller/InterventionalRadiologyController.inl:207]:
```suggestion
```
same

File: src/BeamAdapter/component/controller/InterventionalRadiologyController.h [hunk 3]
```diff
@@ -89,7 +89,6 @@ class InterventionalRadiologyController : public MechanicalStateController<DataT
     ////////////////////// Inherited from BaseObject ///////////////////////////////////////////////
     virtual void init() override ;
     virtual void bwdInit() override ;
-    virtual void reinit() override;
     virtual void draw(const core::visual::VisualParams*) override {}
```
Comment (fredroy) [inline src/BeamAdapter/component/controller/InterventionalRadiologyController.h:92]:
```suggestion
```
Could do the same with draw() was well

Comment (epernod) [reply src/BeamAdapter/component/controller/InterventionalRadiologyController.h:92]: 🙈

**Review by fredroy — APPROVED** [review]

**Pull request merged**